erizo/src/erizo/rtp/SenderBandwidthEstimationHandler.h

```diff
@@ -25,7 +25,7 @@ class SenderBandwidthEstimationHandler : public Handler,
 
  public:
   static const uint16_t kMaxSrListSize = 20;
-  static const uint32_t kStartSendBitrate = 300000;
+  static const uint32_t kStartSendBitrate = 1000000;
```

Comment:
it takes less time to grow, but doesn't affect a poor bandwidth accordingly to my tests.